Bottom: 46f9217888c9acebd399b8bae5da7220e1c019e9
Top:    bdc0ae7702fb6b75959bf3c239104a346303de5a
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-17 22:48:01 +1300

Refresh of copy-stream

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 1d4aff9..4e045a5 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -378,28 +378,34 @@ def domap(*args, **kwargs):
     return list(map(*args, **kwargs))
 
 
-class CopyStreamResult(object):
+class CopyStreamResult(StreamResult):
     """Copies all event it receives to multiple results.
     
     This provides an easy facility for combining multiple StreamResults.
     """
 
     def __init__(self, targets):
+        super(CopyStreamResult, self).__init__()
         self.targets = targets
 
     def startTestRun(self):
+        super(CopyStreamResult, self).startTestRun()
         domap(methodcaller('startTestRun'), self.targets)
 
     def stopTestRun(self):
+        super(CopyStreamResult, self).stopTestRun()
         domap(methodcaller('stopTestRun'), self.targets)
 
     def estimate(self, *args, **kwargs):
+        super(CopyStreamResult, self).estimate(*args, **kwargs)
         domap(methodcaller('estimate', *args, **kwargs), self.targets)
 
     def file(self, *args, **kwargs):
+        super(CopyStreamResult, self).file(*args, **kwargs)
         domap(methodcaller('file', *args, **kwargs), self.targets)
 
     def status(self, *args, **kwargs):
+        super(CopyStreamResult, self).status(*args, **kwargs)
         domap(methodcaller('status', *args, **kwargs), self.targets)
